src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const amount = 40;
+
+  if (days < 3) {
```

Comment:
2. [checklist] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values

and for next lines